source/plugins/languages/visual/libraries/micropython/visual/toolbox.xml

```diff
@@ -0,0 +1,499 @@
+<toolbox>   
+  <category name="ESP8266" colour="#F1C40F">
+    <category name="Pins and GPIO" colour="#6C3483">
```

Comment:
```suggestion
    <category name="GPIO" colour="#6C3483">
```

- The Pin block has only Pin.OUT, this should be a dropdown menu with INPUT and OUTPUT 
- the variable name should be pin (not LED or else)
- add a new category LED that has basically the same blocks, but with different names (eg the mode is always OUTOUT)
-